js/main.js

```diff
@@ -69,12 +90,28 @@ function watchListBtnClick(event) {
   }
   switchView('watchlist-view');
 }
+function portfolioBtnClick(event) {
+  for (var i = 0; i < data.portfolioEntries.length; i++) {
+    if (data.portfolioEntries[i].stockName === data.currentStock) {
+      alert('stocks is already in portfolio');
```

Comment:
**_Never_** use alerts in your code. If it is necessary to show a message to the user, you should use a modal.